config/initializers/omniauth.rb

```diff
@@ -1,6 +1,5 @@
-Rails.application.config.middleware.use OmniAuth::Builder do
-  provider :developer unless Rails.env.production?
-#  provider :twitter, ENV['TWITTER_KEY'], ENV['TWITTER_SECRET']
-  provider :twitter, Figaro.env.twitter_consumer_key, Figaro.env.twitter_consumer_secret
-end
+OmniAuth.config.logger = Rails.logger
 
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :twitter, "your_key", 'api_secret', {:client_options => {:ssl => {:ca_file => Rails.root.join("cacert.pem").to_s}}}
```

Comment:
Line is too long. [126/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.